doc/get-started/basic-concepts.rst

```diff
@@ -249,6 +249,25 @@ dictionary under ``product``:
 This covers scripts and notebooks as tasks, if you want to learn how to use
 functions as tasks, keep scrolling, otherwise, :ref:`skip to the end. <where-to-go-from-here>`
 
+It is also possible to use environment variables in the ``pipeline.yaml`` file. 
```

Comment:
please replace "environment variables" for "placeholders" since that's how we refer to the in the rest of the docs